src/components/ShellView/ShellCell.vue

```diff
@@ -112,12 +115,23 @@ export default {
       this.errorMessage = "";
       this.isLoading = true;
       this.loadingText = "Evaluating query...";
+      let intervalId = setInterval(() => {
+          Axios.get(`/api/cypher/result/${this.cellId}`).then((res) => {
+          if (res.data) {
+            this.loadingText = res.data.finished ? "Processing result..." : `Pipelines Finished: ${res.data.numPipelinesFinished}/${res.data.numPipelines}
+            Current Pipeline Progress: ${Math.round(res.data.pipelineProgress * 100)}%`;
+          } else if (this.loadingText !== "Evaluating query...") {
+              this.loadingText = "Processing results..."
+          }
+        });
+      }, 500);
```

Comment:
For this part, the strings such as `Processing result...` and `Evaluating query...` are reused. I would suggest refactoring them as a enum dictionary.